assets/scss/default.scss

```diff
@@ -0,0 +1,70 @@
+@import "bootstrap";
+
+.row {
+	margin: 20px auto;
+}
+
+figure img, .posters img {
+	max-width: 100%;
+}
+
+hr {
+	border-top: 1px solid #fff;
+}
+
+header {
+	&, a {color: $gray-light;}
+	h1, h2, p {
+		text-transform: uppercase;
+		letter-spacing: 1px;
+	}
+}
+
+h1, h2 {
+	font-size: $font-size-base;
+	padding: 0;
+	margin: 0;
+	line-height: 20px;
+}
+
+.poster {
+    margin: 15px 0 15px 0;
+    line-height: 20px;
+}
+
+.active, header a:hover {
+	background: #fff;
+	color: $gray-base;
+	font-weight: bold;
+}
+
+.bio-photo {
+	margin-bottom: 20px;
+}
+
+// Fitvids
+
+video {
+	max-width: 100%;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered height, max-width